src/Resources/Resources/ChangeLog.md

```diff
@@ -25,6 +25,7 @@
   - `New-AzADServicePrincipal`
   - `Update-AzADServicePrincipal`
 * Ignored Bicep warning message in error stream if exitcode equals zero.
+* Add PrincialType to role assignment requests
```

Comment:
Hi @dagoroz , Please move your change log message to the place under the top of ##upcoming release section. Since current time is our code freeze time, doing this can help us to detect conflicts when release branch merges back to master branch. Thanks.

Reply:
moved